stablehlo/dialect/TypeInference.cpp

```diff
@@ -4458,6 +4551,13 @@ LogicalResult verifyWhileOp(std::optional<Location> location,
         location,
         "expect condition block return a zero-ranked tensor of i1 but got ",
         condReturnTypes[0]);
+  // while_c3
```

Comment:
why do we need that? Itsn't this checked in during type inference?

Reply:
yes.. type inference is checking it. validated.
Removed, kept the test.